source/connect/connection-options/connection-pools.txt

```diff
@@ -57,6 +98,24 @@ secondary nodes, those connection pools grow and there can be
 To support high numbers of concurrent MongoDB requests
 within one process, you can increase ``maxPoolSize``.
 
+The following code creates a ``MongoClient`` instance with a maximum connection
+pool size of ``50`` by specifying the ``maxPoolSize`` option in the
+``options`` object:
+
+.. code-block:: javascript
+
+   const { MongoClient } = require('mongodb');
+
+   const uri = '<connection-string>';
+   const client = new MongoClient(uri, {
+     maxPoolSize: 50
```

Comment:
suggest using a number higher than the default, given that the preceding sentence mentions increasing maxPoolSize.